### ISSUE TITLE
Vertical Line as highlighter 

### ISSUE THREAD

**Issue by arpanda** [deeptools/pyGenomeTracks#322]
Hi,
I tried to use the vlines to highlight a region in the track. 
Here is my ini file setting for the vlines.
```
[vlines]
file=result/peak_0.05.bed
type=vlines
```
Here is a line from the file

```
chr1  4447721 4448374 654 * 4.65  1.85  5.54  -3.69 2.27e-06  0.0067
```
Using this setting, its only highlighting the start region not the entire start to end region. So can you please help me find the solution. 

Also how I can use this feature as python module?

**Comment by lldelisle** [comment]
Hi,
The vertical lines only use the start regions. If you want to have both, you need to have 2 lines in your bed file:
```
chr1  4447721 4447722 654 * 4.65  1.85  5.54  -3.69 2.27e-06  0.0067
chr1  4448373 4448374 654 * 4.65  1.85  5.54  -3.69 2.27e-06  0.0067
```
For the usage as python module, we are working on an api support but this is not ready yet. There is an example here: https://github.com/deeptools/pyGenomeTracks/issues/8 but I don't know if it is still working...

**Comment by arpanda** [comment]
Thanks for the idea and example link.
is there any other way to highlight a region ?

**Comment by lldelisle** [comment]
Not for the moment, but this is a enhancement which could be easy to integrate. I don't have time now but as soon as I have...

**Comment by lldelisle** [comment]
This has been implemented in version 3.7 with option: `type = vhighlight`, see https://github.com/deeptools/pyGenomeTracks/blob/master/pygenometracks/tests/test_data/master_vhighlight.png and ini file: https://github.com/deeptools/pyGenomeTracks/blob/master/pygenometracks/tests/test_data/vhighlight.ini
